distribution/ddf-common/src/main/resources/bin/setenv.bat

```diff
@@ -62,6 +62,11 @@ rem SET KARAF_OPTS=
 rem Uncomment out the line below to enable cxf logging interceptors
 rem set EXTRA_JAVA_OPTS="-Dcom.sun.xml.ws.transport.http.HttpAdapter.dump=true"
 
+set DDF_HOME_PERM=%DDF_HOME:/=\%
+set DDF_HOME_PERM=%DDF_HOME_PERM:\bin\..=\%
+set DDF_HOME_POLICY=/%DDF_HOME%
+set DDF_HOME_POLICY=%DDF_HOME_POLICY:c:=C:%
```

Comment:
❓ Is this assuming DDF_HOME is always in the C drive? What if it ends up in a different drive?